src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -894,26 +883,27 @@ public function exportAction($_format)
             return $productProvider->getCatalogProductList($offset, $limit, $orderBy, $sortOrder, array(), true, false);
         };
 
-        // export CSV
-        $csvTools->exportData(
-            $dataCallback,
-            ['id_product' => 'Product ID',
-                'image_link' => 'Image',
-                'name' => 'Name',
-                'reference' => 'Reference',
-                'name_category' => 'Category',
-                'price' => 'Base price',
-                'price_final' => 'Final price',
-                'sav_quantity' => 'Quantity',
-                'badge_danger' => 'Status',
-                'position' => 'Position',
-            ],
-            100,
-            'product_' . date('Y-m-d_His') . '.csv',
-            30 * 60, // 30 minutes of download max!
-            true // TODO: windows CRLF, to make dynamic or always ON?
+        $translator = $this->get('translator');
+
+        $headersData = array(
+            'id_product' => 'Product ID',
+            'image_link' => $translator->trans('Image', array(), 'Admin.Global'),
+            'name' => $translator->trans('Name', array(), 'Admin.Global'),
+            'reference' => $translator->trans('Reference', array(), 'Admin.Global'),
+            'name_category' => $translator->trans('Category', array(), 'Admin.Global'),
+            'price' => $translator->trans('Base price', array(), 'Admin.Catalog.Feature'),
```

Comment:
We don't have any "base price" in catalog or product page, so we should use the field name it's referring to ("Price (tax excl.) it seems?). Thanks!